cmplibrary/src/main/java/com/sourcepoint/cmplibrary/SpConsentLibImpl.kt

```diff
@@ -316,6 +318,31 @@ internal class SpConsentLibImpl(
         )
     }
 
+    override fun deleteCustomConsentTo(
+        vendors: Array<String>,
+        categories: Array<String>,
+        legIntCategories: Array<String>,
+        successCallback: CustomConsentClient
+    ) {
+        deleteCustomConsentTo(
+            vendors = vendors.toList(),
+            categories = categories.toList(),
+            legIntCategories = legIntCategories.toList(),
+            success = {
```

Comment:
to avoid crashes during the parse, replace the lambda with 
```
success = {
                check { JsonConverter.converter.encodeToString(it) }
                    .executeOnRight { r -> successCallback.transferCustomConsentToUnity(r) }
                    .executeOnLeft {
                        spClient.onError(RuntimeException("An error occurred during delete custom consent request"))
                        pLogger.clientEvent(
                            event = "onError",
                            msg = "An error occurred during delete custom consent request",
                            content = "An error occurred during delete custom consent request"
                        )
                    }
            }
```

Reply:
and please apply the same change to `customConsentGDPR`

Reply:
@Nevazhnovu did you test them manually?